### PR TITLE
Admin Features

### PR DESCRIPTION
**Pull request by farisashai**
# Info

Initial Scaffolding for Admin Home Page

this will be the next biggest set of features to complete once everything user facing is done (can be completely after releasing v2 publicly) - most urgent task admin side will probably be editing and managing store items to upload multiple item images

the goal of this PR is just to set a decent foundation wiring up all links and actions (will need major styling revamps later on)

# Description

- added link to admin settings page on mobile
- added buttons for all possible admin features

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [ ] locally on mobile - use https://ngrok.io to get a copy on a mobile device
- [ ] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have run and passed all new and existing Cypress tests. Add screenshots below.

# Checklist

- [ ] I have performed a self-review of my own code.
- [ ] I have followed the style guidelines of this project.
- [ ] I have documented my code's `src/lib` functions and commented hard to understand areas
      anywhere else.
- [ ] My changes produce no new warnings.

# Screenshots

Please include a screenshot of your Cypress testing suite passing successfully.

If you made any visual changes to the website, please include relevant screenshots below.

### CODE REVIEW

File: src/pages/admin/milestone.tsx
Comment (alexzhang1618) [inline src/pages/admin/milestone.tsx]:
```suggestion
        description="Award points to all active users (e.g. for ACM's 8 year anniversary)"
```

File: src/components/common/LinkButton/style.module.scss
Comment (alexzhang1618) [inline src/components/common/LinkButton/style.module.scss:60]: seeking to learn: why does this need to be `!important`?

**Review by alexzhang1618 — APPROVED** [review]
Changes LGTM but I'd wait for another review before merging 👹

File: src/components/common/LinkButton/style.module.scss
Comment (SheepTester) [reply src/components/common/LinkButton/style.module.scss:60]: because chaining two attribute selectors like `[data-variant='secondary'][data-destructive='true']` appears to have greater precedence than `:disabled`

A way to avoid this could be to set a CSS variable to the background color, then if the button isn't disabled, set the background color to the CSS variable

File: src/pages/admin/milestone.tsx
Comment (SheepTester) [inline src/pages/admin/milestone.tsx:33]: I see that you've copied this from the sign-in page, which is fine, but it might be confusing even for the people using the admin page if it looks too much like the sign-in page. could at least remove the mail/password icons

**Review by SheepTester — APPROVED** [review]
I like that you un-hardcoded the mobile navbar height

File: src/pages/admin/milestone.tsx
Comment (farisashai) [reply src/pages/admin/milestone.tsx:33]: this was very extremely temporary i agree we shouldn't use this specific component here

**Pull request merged**
